certified-connectors/Connect2All/readme.md

```diff
@@ -0,0 +1,57 @@
+## GAC Connect2All
+GAC Connect2All integrates the PowerAutomate platform to Business Central and makes adaptations and the extensibility to connect virtual any Datasource to your business processes in Business Central possible. You do not need to be a Professional developer to seamlessly extend Business Central to interact to other business application and Datasources. GAC Connect2All delivers your business the scalability and flexibility to create smart integration solutions you need today and tomorrow.
```

Comment:
>integrates the PowerAutomate platform to Business Central

Could this please be changed to say "integrates the Microsoft Power Platform to..."

Reply:
I updated this text as suggested.